resources/file.go

```diff
@@ -110,6 +130,37 @@ func (obj *FileRes) Validate() error {
 	return obj.BaseRes.Validate()
 }
 
+// mode return the file permission specified on the graph. It doesn't handle the
+// case where the mode is not specified. The caller should check obj.Mode is not
+// empty.
+func (obj *FileRes) mode() (os.FileMode, error) {
+	m, err := strconv.ParseInt(obj.Mode, 8, 32)
+	if err != nil {
+		return os.FileMode(0), errwrap.Wrapf(err, "Mode should be an octal number (%s)", obj.Mode)
+	}
+	return os.FileMode(m), nil
+}
+
+// uid returns the user id for the owner specified in the graph. Caller should
```

Comment:
function comments are sentences, and should end with a period. Also, "the owner specified in the graph" -- isn't this the owner specified by the file?